util/missingmacro.txt

```diff
@@ -69,6 +69,7 @@ ENGINE_load_capi(3)
 ENGINE_load_afalg(3)
 ENGINE_load_cryptodev(3)
 ENGINE_load_rdrand(3)
+ENGINE_load_rndr(3)
```

Comment:
This would need to be documented if it weren't going to be removed :)

Reply:
removed thus fixed